app/Libraries/BeatmapsetDiscussionReview.php

```diff
@@ -105,14 +105,17 @@ public static function create(Beatmapset $beatmapset, array $document, User $use
             foreach ($document as $block) {
                 switch ($block['type']) {
                     case 'paragraph':
-                        // escape embed injection attempts
-                        $text = preg_replace('/%\[\]\(#(\d+)\)/', '%\[\]\(#$1\)', $block['text']);
-                        $output[] = "{$text}\n";
+                        array_push($output, [
+                            'type' => 'paragraph',
+                            'text' => $block['text'],
```

Comment:
Yesterday I didn't realize this is part of this PR but this should be cast to string otherwise weird stuff happens when it's rendered.